### ISSUE TITLE
自定义损失函数。（Custom loss function.）自定义损失函数的输入参数（Custom loss input）。

### ISSUE THREAD

**Issue by zxccvdql** [open-mmlab/mmsegmentation#2948]
`def my_loss(pred, target):`
`    assert pred.size() == target.size() and target.numel() > 0`
`    loss = torch.abs(pred - target)`
`    return loss`

这是文档中的自定义损失函数，损失函数只能输入`pred, target`。想在损失函数中添加模型参数作为输入，我该如何设置？

Custom loss function can only take `pred` and `target` inputs. Cannot take model or other parameters.
